pkg/render/logstorage.go

```diff
@@ -430,6 +420,12 @@ func (es elasticsearchComponent) podTemplate() corev1.PodTemplateSpec {
 	// https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-managing-compute-resources.html and
 	// https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-jvm-heap-size.html#k8s-jvm-heap-size
 
+	var volumeMounts []corev1.VolumeMount
+	if es.dexCfg != nil {
+		volumeMounts = []corev1.VolumeMount{
```

Comment:
Could you please add a RequiredVolumeMounts() function to the dexCfg that returns this instead.